pkgs/development/libraries/science/math/tensorrt/generic.nix

```diff
@@ -25,16 +26,14 @@ stdenv.mkDerivation {
 
   nativeBuildInputs = [
     autoPatchelfHook
+    autoAddOpenGLRunpathHook
   ];
 
   # Used by autoPatchelfHook
   buildInputs = [
     cudatoolkit.cc.cc.lib # libstdc++
-  ];
-
-  propagatedBuildInputs = [
     cudatoolkit
-    cudaPackages.cudnn
+    cudnn
```

Comment:
Is there a reason not to use the cudaPackages version?

Reply:
If I understood @SomeoneSerge's feedback on my earlier commits, this is the convention that CUDA packages should follow.